src/car/Convertible.java

```diff
@@ -0,0 +1,67 @@
+package car;
+
+import java.util.Scanner;
+
+public class Convertible extends Car implements Movable {
+    private static final int MAX_ALLOWED_SPEED = 200;
+    private static final int MIN_PRICE = 1000;
+    private static final int MAX_PRICE = 40000000;
+    private int price;
+    static Scanner scanner = new Scanner(System.in);
+
```

Comment:
static is bad for scanner. Scanner can be closed and in this case all instances of Convertible will be broken

Reply:
Thanks! I will improve my code